config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  get 'static_pages/home'
```

Comment:
Trong ruby ưu tiển sử dụng " "  nháy đôi nhé. file css và js thì dùng nháy đơn nha.. :D 
```suggestion
  get "static_pages/home"
```
tương tự ở dưới nha.